Lettering error

Getting a traceback while playing around with a new font. I suspect it's telling me there's a missing glyph (I probably accidentally moved one to another's layer), and I'm not sure just how much work we need to put in on errors that will (one assumes) only show up during font development, but anyway here it is for reference.

Traceback (most recent call last):
  File "lib/extensions/lettering.py", line 414, in apply
  File "lib/extensions/lettering.py", line 355, in update_lettering
  File "lib/lettering/font.py", line 195, in render_text
  File "lib/lettering/font.py", line 107, in _load_variants
  File "lib/lettering/font_variant.py", line 61, in __init__
  File "lib/lettering/font_variant.py", line 77, in _load_glyphs
  File "lib/lettering/glyph.py", line 40, in __init__
  File "lib/lettering/glyph.py", line 86, in _process_bbox
ValueError: min() arg is an empty sequence

(If there's an easy way to detect an empty layer, I'm all ears.)

Oooooh. That release doesn't seem to work at all though:

Opening Lettering gets:
```11:54:49: can't open file '/home/silver/.config/inkscape/extensions/inkstitch/bin/_internal/icons/inkstitch256x256.png' (error 2: No such file or directory)
11:54:49: Failed to load image from file "/home/silver/.config/inkscape/extensions/inkstitch/bin/_internal/icons/inkstitch256x256.png".
```

and then switching to the font gets:

```12:00:02: can't open file '/home/silver/.config/inkscape/extensions/inkstitch/bin/_internal/icons/slower.png' (error 2: No such file or directory)
12:00:02: Failed to load image from file "/home/silver/.config/inkscape/extensions/inkstitch/bin/_internal/icons/slower.png".```

(both in the new error type)

and then ignoring the lack of preview and hitting Apply gets

```Traceback (most recent call last):
  File "lib/gui/simulator.py", line 1138, in refresh_simulator
  File "lib/gui/simulator.py", line 997, in __init__
  File "lib/gui/simulator.py", line 959, in __init__
  File "lib/debug.py", line 231, in decorated
  File "lib/gui/simulator.py", line 57, in __init__
  File "lib/gui/simulator.py", line 283, in load_icon
wx._core.wxAssertionError: C++ assertion ""IsOk()"" failed at /home/wxpy/wxPython-4.2.1/ext/wxWidgets/src/common/image.cpp(442) in Scale(): invalid image

Traceback (most recent call last):
  File "wx/core.py", line 3427, in <lambda>
  File "lib/gui/simulator.py", line 1154, in refresh_simulator
AttributeError: 'NoneType' object has no attribute 'Show'``` in the native error format.

The good news: it *does* create the Lettering object (or at least creates "_luenesia Test" - gotta go check that capital B, I guess).

(The other bad news: it locked up LXDE harrrrrrd the second time, I had to reboot. This is probably not a new problem though; I am not sure if it's my keyboard driver, LXDE, or Inkscape. Usually Inkscape gloms an entire core and never returns, but this time ssh-ing in and killall inkscape didn't work, nor was anything claiming to be using much CPU *or* memory, but the window manager gradually stopped responding to anything. Gonna switch window managers someday and see if it fixes it, but not today.)

--- a/lib/lettering/font_variant.py
+++ b/lib/lettering/font_variant.py
@@ -75,7 +75,7 @@ class FontVariant(object):
                 glyph_name = layer.attrib[INKSCAPE_LABEL]
                 try:
                     self.glyphs[glyph_name] = Glyph(layer)
-                except AttributeError:
+                except (AttributeError, ValueError):
                     pass
 
     def _get_variant_file_paths(self):

